.github/workflows/build.yml

```diff
@@ -325,18 +325,6 @@ jobs:
         if: matrix.parallel-parts == 'functionaltests-postgres'
         run: bin/phpunit --colors --stop-on-failure -c Build/BuildEssentials/PhpUnit/FunctionalTests.xml --testsuite "Neos tests" --verbose
 
-      - name: Run behat tests (PGSQL)
-        if: matrix.parallel-parts == 'functionaltests-postgres'
-        run: |
-          FLOW_CONTEXT=Testing/Behat ./flow configuration:show
-          FLOW_CONTEXT=Testing/Behat ./flow doctrine:migrationstatus
-
-          FLOW_CONTEXT=Testing/Behat ./flow behat:setup
-          FLOW_CONTEXT=Testing/Behat ./flow doctrine:create
-          FLOW_CONTEXT=Testing/Behat ./flow doctrine:migrationversion --add --version all
-          # TODO: RE-ENABLE AFTER FIXING
-          #bin/behat --stop-on-failure -f progress -c Packages/Neos/Neos.Neos/Tests/Behavior/behat.yml.dist --tags ~@browser
-
```

Comment:
Why? This was added at some point, to make sure minimal divergences in DB behaviour don't caused bugs we hardly notice. If you simply want to make the builds faster, I disagree…

Reply:
the tests are actually disabled currently see `#` and are run via `composer test:behavioral`.

I dont know about our postgres status but once we implemented the postgres adapter we will run also `test:behavioral` with postgres.

Reply:
Issue for that followup task https://github.com/neos/neos-development-collection/issues/3855